eclair-core/src/main/scala/fr/acinq/eclair/blockchain/electrum/ElectrumClient.scala

```diff
@@ -381,6 +380,9 @@ object ElectrumClient {
   case class BroadcastTransaction(tx: Transaction) extends Request
   case class BroadcastTransactionResponse(tx: Transaction, error: Option[Error]) extends Response
 
+  case class GetTransactionIdFromPosition(height: Int, tx_pos: Int, merkle: Boolean = false) extends Request
+  case class GetTransactionIdFromPositionResponse(txid: ByteVector32, merkle: List[ByteVector32]) extends Response
```

Comment:
```suggestion
  case class GetTransactionIdFromPositionResponse(txid: ByteVector32, merkle: Seq[ByteVector32]) extends Response
```

Better use the interface than the implementation (but I see that we haven't been consistent in this file).

Reply:
Agreed!